contracts/token/ERC721/extensions/ERC721Consecutive.sol

```diff
@@ -19,12 +19,12 @@ import {Checkpoints} from "../../../utils/structs/Checkpoints.sol";
  * Using this extension removes the ability to mint single tokens during contract construction. This ability is
  * regained after construction. During construction, only batch minting is allowed.
  *
- * IMPORTANT: This extension bypasses the hooks {_beforeTokenTransfer} and {_afterTokenTransfer} for tokens minted in
- * batch. The hooks will be only called once per batch, so you should take `batchSize` parameter into consideration
- * when relying on hooks.
+ * IMPORTANT: This extension does not call the {_update} function for tokens minted in batch. Any logic added to this
+ * function through overrides will not be triggered when token are minted in batch. You may want to also override
+ * {_increaseBalance} or {_mintConsecutive} to account for these mints.
  *
- * IMPORTANT: When overriding {_afterTokenTransfer}, be careful about call ordering. {ownerOf} may return invalid
- * values during the {_afterTokenTransfer} execution if the super call is not called first. To be safe, execute the
+ * IMPORTANT: When overriding {_mintConsecutive}, be careful about call ordering. {ownerOf} may return invalid
+ * values during the {_mintConsecutive} execution if the super call is not called first. To be safe, execute the
  * super call before your custom logic.
```

Comment:
This comment referred to `_afterTokenTransfer` because of how burning was implemented. I think it should now refer to `_update`.

```suggestion
 * IMPORTANT: When overriding {_update}, be mindful about call ordering. {ownerOf} may return invalid
 * values during {_update} execution if `ERC721Consecutive`'s code is not executed first. To be safe, make sure
 * `ERC721Consecutive._update` fully executes before your custom logic, and test for the expected behavior.
```

Reply:
I was really meaning `_mintConsecutive`. I guess this is true of both functions, buth since we this comment is in `ERC721Consecutive`, I think we should focus on the consecutive minting part.

Maybe we want another similar warning in `ERC721`.

In both cases I think the reasonning is the same: if you put logic before calling super, you'll be executing that logic in a context where the change have not been applied. Tokens are still owned by the previous owner, and balances have not been updated. It feels obvious, maybe we don't need a warning at all.